### DISCUSSION TITLE
Release notes readability

### DISCUSSION THREAD

**Discussion by morriq** [vercel/next.js#21992] [Ideas]
Hello,

I started to upgrade packages in some project and I use from vercel: 

@next/bundle-analyzer
@next/eslint-plugin-next
next

but I would like to know what's changed to prevent some issues in application.

That's why I land here https://github.com/vercel/next.js/releases/tag/v10.0.6 and I think it's painful to read it :(. Maybe consider changing presentation of it? For example babel solved it nicely in monorepo: https://github.com/babel/babel/blob/main/CHANGELOG.md and https://github.com/babel/babel/releases/tag/v7.12.15

**Comment by leethree** [comment]
agreed. it's very difficult to figure out what's changed with items like "fix: apply correct fix: "

and there are so many `canary` releases in the release page. would be great to have a human-readable `CHANGELOG.md`. 

see https://github.com/vercel/next.js/discussions/11991

**Reply by tusbar** [reply]
Reviving this thread. 

A lot of people are using tools like dependabot/renovate, a nextjs package change log is painful to read, it's just a list of merged PR, often with approximate or bad naming, for example:

> Revert "Revert "Revert "Revert "Initial support for metadata (#44729)"" (#45113)"": #45196

This makes it cumbersome, and I often have to go through a lot of the PRs to understand what changes, and where. 

Also, the same release notes are shown for any next package update, may it be `@next/font` or `next`, or the eslint plugin…

I'd suggest keeping a separate change log for each package, maybe put an end to the automated release notes, or rework them a little. It requires more work, but Vercel can probably afford this, no?
